config/src/credentials.rs

```diff
@@ -276,10 +286,14 @@ async fn send_http_request(
     payload: serde_json::Value,
     header_map: HeaderMap,
     disable_cert_verification: bool,
+    tls_certs: &[u8],
 ) -> Result<Token, reqwest::Error> {
+    let cert = Certificate::from_pem(tls_certs).expect("parse pem file");
```

Comment:
Will this work if the env var pravega_client_tls_cert_path points to a file with multiple certificates or a directory with multiple certificates?

Reply:
yes it should be. The tls_certs contains all the certs in a byte array format and `Certificate::from_pem` should extract mutiple certs.